models.py

```diff
@@ -285,25 +318,33 @@ def _compute_emissions(self,
         assert prefixes is None or prefixes.size() == (batch_size, seq_length, 2)
         assert suffixes is None or suffixes.size() == (batch_size, seq_length, 2)
```

Comment:
Put asserts for `mask` after this line please.

Reply:
added